src/layout/DefaultLayout/DefaultLayout.tsx

```diff
@@ -27,7 +29,11 @@ const DefaultLayout = ({
     { path: '/ussf-documentation', label: 'USSF documentation' },
   ]
 
-  return (
+  const { data } = useGetUserQuery()
```

Comment:
Suggestion(nit-pick): Maybe this would be better renamed `user` instead of `data`?